kube_scheduler/tests/test_kube_scheduler_1_26.py

```diff
@@ -0,0 +1,132 @@
+# (C) Datadog, Inc. 2023-present
```

Comment:
The comments I would have here are the same as the ones I have for [your other PR](https://github.com/DataDog/integrations-core/pull/15731#pullrequestreview-1615932692)